schema/asset_uploads/create_asset_request_mutation.js

```diff
@@ -3,6 +3,52 @@
 import { GraphQLString, GraphQLNonNull, GraphQLObjectType } from "graphql"
 import { mutationWithClientMutationId } from "graphql-relay"
 
+export const S3PolicyConditionsType = new GraphQLObjectType({
+  name: "S3PolicyConditionsType",
+  description: "The conditions for uploading assets to media.artsy.net",
+  fields: {
+    bucket: {
+      description: "The bucket to upload to.",
+      type: new GraphQLNonNull(GraphQLString),
+    },
+    acl: {
+      description: "The assigned access control",
+      type: new GraphQLNonNull(GraphQLString),
+    },
+    gemini_key: {
+      description: "A key which is prefixed on your file",
+      type: new GraphQLNonNull(GraphQLString),
+    },
+    success_action_status: {
+      description: "The returned status code, currently always 201",
+      type: new GraphQLNonNull(GraphQLString),
+    },
+  },
+})
+
+export const S3PolicyDocumentType = new GraphQLObjectType({
+  name: "S3PolicyDocumentType",
+  description: "An policy for uploading assets to media.artsy.net",
+  fields: {
+    expiration: {
+      description: "An expiration date string.",
+      type: new GraphQLNonNull(GraphQLString),
+    },
+    conditions: {
+      description: "The details for the upload",
+      type: new GraphQLNonNull(S3PolicyConditionsType),
+      resolve: ({ conditions }) => {
+        return {
+          bucket: conditions[0].bucket,
+          acl: conditions[2].acl,
+          success_action_status: conditions[3].success_action_status,
+          gemini_key: conditions[1][2],
+        }
```

Comment:
Takes the data which is structured as an S3 policy array, and turns it into more structured data